modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Execution.scala

```diff
@@ -77,6 +77,10 @@ object Execution {
 sealed trait Result
 
 object Result {
-  case class OK[R](r: R) extends Result
+  case class OK(response: Response) extends Result
   case class Error[E](e: E) extends Result
+
+  sealed trait Response
+  case class Configured(r: String) extends Response
+  case class Observed(fileId: FileId) extends Response
```

Comment:
I'm confused. That `Response` is not a UI event, it's the result of action that's launched asynchronously by the engine that when completed is processed again by the engine to create a new application state that is sent to the UI together with a UI event.

I think what you mean is that you need the UI event `StartObserve` here: https://github.com/gemini-hlsw/ocs3/blob/develop/modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala#L12

Sorry to be pestering you, but I need to be sure we are in the same page before going ahead.